jq.sh

```diff
@@ -1,6 +1,6 @@
 package: jq
 version: v1.6
```

Comment:
unsure ... what we need to put as "version"?

Reply:
`v1.6-alice1`, I would say.